Lib/pathlib.py

```diff
@@ -664,7 +664,7 @@ def is_absolute(self):
         # ntpath.isabs() is defective - see GH-44626 .
         if self._flavour is ntpath:
             return bool(self.drive and self.root)
```

Comment:
Hah! I was wondering why I couldn't reproduce any speedup on my machine for `Path.is_absolute()`, and really puzzling over it. Then I realised that on my machine, we never even get to the line you're optimising, because I'm on Windows :)

Reply:
I guess I'll have to take your word for it on this one that this really does improve performance ;)

Reply:
I can at least try to explain what's going on here :)

If we call `isabs(self)`, the call tree looks like:

- `isabs(path)`
  - `os.fspath(path)`
    - `path.__fspath__()`
      - `path.__str__()` <-- somewhat slow
        - `path.drive`, `path.root`, `path._tail`
          - `path._load_parts()`
            - `path._parse_path()` <-- slow  

The thing is, path normalization makes no difference when determining if a POSIX path is absolute! Normalized or not, an absolute path always starts with `/`, and a relative path does not.